packages/oidc/src/refresh/refreshGrant.ts

```diff
@@ -26,11 +26,22 @@ import {
   IIssuerConfig,
   KeyPair,
   TokenEndpointResponse,
+  DEFAULT_SCOPES,
 } from "@inrupt/solid-client-authn-core";
 // NB: once this is rebased on #1560, change dependency to core package.
 import formUrlEncoded from "form-urlencoded";
 import { validateTokenEndpointResponse } from "../dpop/tokenExchange";
 
+// Camelcase identifiers are required in the OIDC specification.
+/* eslint-disable camelcase */
+
+interface IRefreshRequestBody {
+  grant_type: "refresh_token";
+  refresh_token: string;
+  scope: typeof DEFAULT_SCOPES;
```

Comment:
I think this is too strict? i.e., should it be possible to refresh with explicitly different scopes?

Reply:
Currently we're not giving any flexibility to control the scopes, so I don't think we need to be less strict here. Since this is only internal, it will be easy to change if required later.